src/components/Customizer/Customizer.tsx

```diff
@@ -1,18 +1,25 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 import '../../yamui';
 import * as React from 'react';
-import {
-  Customizer as FabricCustomizer,
-  ICustomizerProps as CustomizerProps,
-} from 'office-ui-fabric-react/lib/Utilities';
+import { Customizer as FabricCustomizer, ICustomizerProps } from 'office-ui-fabric-react/lib/Utilities';
+import { BaseComponentProps } from '../../util/BaseComponent/props';
 import { createTheme, ITheme } from 'office-ui-fabric-react';
 
-export { createTheme, ITheme, CustomizerProps };
+export { createTheme, ITheme, ICustomizerProps };
+
+export interface CustomizableComponentProps extends BaseComponentProps {
+  /**
+   * Theme used to style the component. This will often be provided by `customizable`.
+   */
+  theme?: ITheme;
+}
```

Comment:
I moved this from `src/util/BaseComponent/props.ts` because I wanted it to be in the same place as the `nullTheme`, and `nullTheme` didn't seem to fit well in `props.ts`.

Reply:
*defaultTheme